index.html

```diff
@@ -4430,9 +4548,7 @@ <h2 id="AudioBuffer">
           <a><code>OfflineAudioContext</code></a> and an
           <a><code>AudioContext</code></a>.
         </p>
-        <dl title=
-        "[Constructor(&lt;a&gt;AudioBufferOptions&lt;a&gt; options)]interface AudioBuffer"
-        class="idl">
+        <dl title="interface AudioBuffer" class="idl">
```

Comment:
I think you forget to define the actual constructor for this.  And the constructor for this doesn't follow the AudioNode constructors because context is not the first arg.  Not sure where the best place to describe the constructor should be.